experimental/PropertyDDS/packages/property-dds/package.json

```diff
@@ -61,7 +61,7 @@
     "@fluidframework/local-driver": ">=2.0.0-internal.1.3.0 <2.0.0-internal.2.0.0",
     "@fluidframework/mocha-test-setup": ">=2.0.0-internal.1.3.0 <2.0.0-internal.2.0.0",
     "@fluidframework/sequence": ">=2.0.0-internal.1.3.0 <2.0.0-internal.2.0.0",
-    "@fluidframework/server-local-server": "^0.1038.1000",
+    "@fluidframework/server-local-server": "^0.1037.2000",
```

Comment:
Shouldn't this be 1037.2001?

Reply:
On a clean install this will resolve to 0.1037.2001, but you are right if your nodemodules already has 0.1037.2000 (somehow) install will be happy as well. I'll play it safe and bump.